ui/favicon.svg

```diff
@@ -0,0 +1 @@
.cls-1,.cls-10,.cls-3,.cls-4,.cls-5,.cls-7{stroke-linecap:round;}.cls-1,.cls-10,.cls-5,.cls-7{stroke-width:3px;}.cls-1,.cls-10,.cls-2,.cls-3,.cls-4,.cls-5,.cls-6,.cls-7{fill-rule:evenodd;}.cls-3,.cls-4,.cls-5,.cls-6{fill:#fff;}.cls-3{stroke-width:2.91px;}.cls-4{stroke-width:2.82px;}.cls-10,.cls-7{fill:#f6d2a2;}.cls-7{stroke:#231f20;}.cls-8{fill:#febd15;}.cls-9{font-size:130.48px;font-family:Arial-BoldMT, Arial;font-weight:700;}</style></defs><title>ddnsgopher.icon</title><path class="cls-1" d="M27.31,93.21C-21.42,79.51,14.81,17.51,54,43Z"/><path class="cls-1" d="M243.83,38.36c38.57-27.09,73.43,34,28.6,49.42Z"/><path class="cls-2" d="M29.34,76.09c-6.43-3.4-11.14-8-7.21-15.46C25.77,53.75,32.53,54.5,39,57.9Z"/><path class="cls-2" d="M262.27,69.62c6.43-3.4,11.15-8,7.21-15.45-3.64-6.89-10.4-6.14-16.83-2.74Z"/><path class="cls-1" d="M286.18,300.15c.34-23.18-2.14-47.14-3.42-69-2.1-35.74.3-72-3.43-107.23-.21-2,4.06-5.13,3.81-7.14-.22-1.71-4.95-2.29-5.2-4A197.91,197.91,0,0,0,265.7,66.46c-18.2-35.8-49.18-48.33-84.54-52.25-3.11-.34-3.88-4.71-7-4.93-2.08-.15-6.55,3.82-8.65,3.72-6-.28-12-.39-18.12-.39-24.14,1.91-47.5,5.51-67.46,14.08-3.81,1.63-9.76-1.51-13.31.51-3.15,1.8-3.94,8.72-6.86,10.86-14,10.27-25.13,24.63-31.93,44.86-2.76,8.23,1.52,21-5.94,24.76-12,6.15-1.55,12.56-1.87,18.86-1.9,38,6.3,76.48,4.75,114.88-.66,16.19-3.48,40.24-4.54,58.73"/><path class="cls-3" d="M156,78.1c8.76,50.92,92,37.46,80-13.89C225.27,18.15,153,30.9,156,78.1"/><path class="cls-4" d="M56.49,86.34c11.35,44.33,82.33,33,79.65-11.49-3.2-53.25-90.5-43-79.65,11.49"/><ellipse cx="150.96" cy="159.38" rx="25.39" ry="32.86"/><path class="cls-5" d="M166,139.21c0,6.62,1.5,14,.25,21.09-1.69,3.2-5,3.53-7.87,4.83a11.59,11.59,0,0,1-8.86-6.92c-1-8,.38-15.82.64-23.86Z"/><ellipse cx="98.47" cy="77.62" rx="11.99" ry="12.98"/><path class="cls-6" d="M104.62,77.34c0,3.36-2.33,6.08-5.21,6.08s-5.2-2.72-5.2-6.08,2.33-6.08,5.2-6.08S104.62,74,104.62,77.34Z"/><ellipse cx="198.89" cy="74.38" rx="11.79" ry="12.98"/><path class="cls-6" d="M206.22,74.21c0,4.15-2.83,7.51-6.32,7.51s-6.32-3.36-6.32-7.51,2.83-7.5,6.32-7.5S206.22,70.07,206.22,74.21Z"/><path class="cls-5" d="M131.75,138.49c-5.23,12.67,2.91,38,17.1,19.32-1-8,.38-15.82.63-23.86Z"/><path class="cls-7" d="M133.54,114.08c-9.75.83-17.72,12.42-12.65,21.59,6.71,12.14,21.69-1.08,31,.16,10.73.22,19.53,11.35,28.16,2,9.59-10.39-4.13-20.5-14.86-25Z"/><path class="cls-2" d="M132.31,113.57c-.72-16.88,31.47-19,35.27-4.86s-33.69,17.38-35.27,4.86Z"/><polygon id="lighting_bolt" data-name="lighting bolt" class="cls-8" points="10.65 292.5 171.49 238.88 150.71 214.26 295.68 150.26 190.09 219.74 219.64 250.92 10.65 292.5"/><text class="cls-9" transform="translate(15.68 289.59)">DNS</text><path class="cls-10" d="M28.18,222.19s10.24,19.32.26,18.47c-3.07,12.48-15.52,1.79-15.52,1.79s-4.76-5.64-5-12.91C7.61,219.53,19.84,209.64,28.18,222.19Z"/><path class="cls-10" d="M289,218.12s-21.86-.54-16.65,8c-9.79,8.31,5.33,14.68,5.33,14.68s5.22,3.92,13.78-1.3S302.25,223.86,289,218.12Z"/></svg>
```

Comment:
Because it's gonna be tiny, I reduced its size using [svgomg](https://jakearchibald.github.io/svgomg/) 😉 

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" id="レイヤー_1" data-name="レイヤー 1" viewBox="0 0 300 300"><defs><style>.cls-1{fill:#6edbd1}.cls-1,.cls-10,.cls-5{stroke:#000;stroke-linecap:round;stroke-width:3px}.cls-1,.cls-10,.cls-2,.cls-5,.cls-6{fill-rule:evenodd}.cls-5,.cls-6{fill:#fff}.cls-10{fill:#f6d2a2}</style></defs><path d="M27 93c-48-13-12-75 27-50zM244 38c38-27 73 34 28 50z" class="cls-1"/><path d="M29 76c-6-3-11-8-7-15s11-6 17-3zM262 70c7-4 11-8 7-16-3-7-10-6-16-3z" class="cls-2"/><path d="M286 300c1-23-2-47-3-69-2-36 0-72-4-107l4-7c0-2-5-3-5-4a198 198 0 00-12-47c-19-35-49-48-85-52-3 0-4-5-7-5l-8 4h-19c-24 2-47 5-67 14-4 1-10-2-13 0-4 2-4 9-7 11a89 89 0 00-32 45c-3 8 1 21-6 25-12 6-2 12-2 19-2 38 6 76 5 114l-5 59" class="cls-1"/><path fill="#fff" fill-rule="evenodd" stroke="#000" stroke-linecap="round" stroke-width="2.9" d="M156 78c9 51 92 38 80-14-11-46-83-33-80 14"/><path fill="#fff" fill-rule="evenodd" stroke="#000" stroke-linecap="round" stroke-width="2.8" d="M56 86c12 45 83 33 80-11-3-53-90-43-80 11"/><ellipse cx="151" cy="159.4" rx="25.4" ry="32.9"/><path d="M166 139c0 7 2 14 0 21-1 4-5 4-8 5a12 12 0 01-8-7v-24z" class="cls-5"/><ellipse cx="98.5" cy="77.6" rx="12" ry="13"/><path d="M105 77c0 4-3 6-6 6s-5-2-5-6 3-6 5-6 6 3 6 6z" class="cls-6"/><ellipse cx="198.9" cy="74.4" rx="11.8" ry="13"/><path d="M206 74c0 4-3 8-6 8s-6-4-6-8 2-7 6-7 6 3 6 7z" class="cls-6"/><path d="M132 138c-5 13 3 38 17 20v-24z" class="cls-5"/><path fill="#f6d2a2" fill-rule="evenodd" stroke="#231f20" stroke-linecap="round" stroke-width="3" d="M134 114c-10 1-18 12-13 22 7 12 22-1 31 0 11 0 19 11 28 2 10-11-4-21-15-25z"/><path d="M132 114c0-17 32-19 36-5s-34 17-36 5z" class="cls-2"/><path id="lighting_bolt" fill="#febd15" d="M11 293l160-54-20-25 145-64-106 70 30 31-209 42z" data-name="lighting bolt"/><text font-family="Arial-BoldMT,Arial" font-size="130.5" font-weight="700" transform="translate(16 290)">DNS</text><path d="M28 222s10 20 0 19c-3 12-15 1-15 1s-5-5-5-12c0-10 12-20 20-8zM289 218s-22 0-17 8c-9 8 6 15 6 15s5 4 13-1 11-16-2-22z" class="cls-10"/></svg>ss="cls-5" d="M131.75,138.49c-5.23,12.67,2.91,38,17.1,19.32-1-8,.38-15.82.63-23.86Z"/><path class="cls-7" d="M133.54,114.08c-9.75.83-17.72,12.42-12.65,21.59,6.71,12.14,21.69-1.08,31,.16,10.73.22,19.53,11.35,28.16,2,9.59-10.39-4.13-20.5-14.86-25Z"/><path class="cls-2" d="M132.31,113.57c-.72-16.88,31.47-19,35.27-4.86s-33.69,17.38-35.27,4.86Z"/><polygon id="lighting_bolt" data-name="lighting bolt" class="cls-8" points="10.65 292.5 171.49 238.88 150.71 214.26 295.68 150.26 190.09 219.74 219.64 250.92 10.65 292.5"/><text class="cls-9" transform="translate(15.68 289.59)">DNS</text><path class="cls-10" d="M28.18,222.19s10.24,19.32.26,18.47c-3.07,12.48-15.52,1.79-15.52,1.79s-4.76-5.64-5-12.91C7.61,219.53,19.84,209.64,28.18,222.19Z"/><path class="cls-10" d="M289,218.12s-21.86-.54-16.65,8c-9.79,8.31,5.33,14.68,5.33,14.68s5.22,3.92,13.78-1.3S302.25,223.86,289,218.12Z"/></svg>
```

Reply:
Although maybe it's better to keep your original version which is easier to re-model... And saving 1KB on a local network won't change anything really I guess 😄

Reply:
If you decide to use the favicon.svg directly as favicon yeah why not minifying it (even though that might indeed be overkill ^^). But if we keep the .ico let's keep the svg not minimified. Just one suggestion if we keep the minimified version: have a look at it when scaled up to the size of a phone icon to make sure minification didn't mess with the icon, sometimes paths are removed in the process

Reply:
Oh silly me I didn't see the favicon.ico file in the files added; let's forget about this suggestion.

1. Can we make the favicon.ico smaller, like 1-2KB as it's usually just a few pixels ~ 64x64? Or maybe you would see a reason to keep it high res?
2. The SVG `id="レイヤー_1" data-name="レイヤー 1"`: I have nothing against UTF-8 characters, but WhY 😄?

Reply:
I haven't edited the id, it's the original one from https://raw.githubusercontent.com/golang-samples/gopher-vector/master/gopher.svg

Reply:
Cool thanks for taking the time to explain 👍 !